src/main/java/com/example/pfairplayservice/jpa/model/MemberEntity.java

```diff
@@ -30,6 +27,12 @@
     @GenericGenerator(name = "member-uid", strategy = "uuid")
     private String uid;
 
+    @ManyToMany
+    @JoinTable(name = "memberEntityTeamEntity"
+            , joinColumns = {@JoinColumn(name = "memberEntityUid")}
+            , inverseJoinColumns = {@JoinColumn(name = "teamEntityTid")})
```

Comment:
Why not `memberUid` and `teamTid`?

Reply:
I haven't found solution about applying ManyToMany relation on jpa.
I'm still working on it.
if you have any reference, please share to me.